pkg/cloudprovider/providers/openstack/openstack_loadbalancer.go

```diff
@@ -626,6 +631,26 @@ func (lbaas *LbaasV2) EnsureLoadBalancer(clusterName string, apiService *v1.Serv
 	floatingPool := getStringFromServiceAnnotation(apiService, ServiceAnnotationLoadBalancerFloatingNetworkId, lbaas.opts.FloatingNetworkId)
 	glog.V(4).Infof("EnsureLoadBalancer using floatingPool: %v", floatingPool)
 
+	var internalAnnotation bool
+	serviceAnnotationLoadBalancerInternal := getStringFromServiceAnnotation(apiService, ServiceAnnotationLoadBalancerInternal, "true")
+	switch serviceAnnotationLoadBalancerInternal {
+	case "true":
+		internalAnnotation = true
+		glog.V(4).Infof("Ensure an internal loadbalancer service with ServiceAnnotationLoadBalancerInternal annotation: %v",
```

Comment:
avoid variable name in the user visible log - use the actual annotation they should use